package.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "cornercutter",
+	"version": "0.1.0",
+	"private": true,
+	"scripts": {
+		"tauri": "tauri",
+		"client": "tauri dev",
+		"dev": "npm run dev --prefix frontend",
+		"build": "npm run build --prefix frontend"
+	},
+	"dependencies": {
+		"@mantine/core": "^4.2.11",
```

Comment:
have you used mantine before? just checking in case we should consider alternative libs ala chakra or similar - if you're already comfortable with this lib tho no issue

Reply:
I actually haven't, I just liked the style, however, Chakra definitely has a better range of components (And tabs, which will be useful for this!) so I'm happy to go with that :)